docs/scos/dev/guidelines/monitorable-process-guidelines.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Monitorable process guidelines
+description: @TODO
+last_updated: @TODO
+template: concept-topic-template
+related:
+- title: NFR guidelines
+  link: docs/scos/dev/guidelines/nfr-guidelines.html
+- title: Operatable feature guidelines
+  link: docs/scos/dev/guidelines/operatable-feature-guidelines.html
+- title: Process documentation guidelines
+  link: docs/scos/dev/guidelines/process-documentation-guidelines.html
+---
+
+In order to enable Operations Team to track/correlate issues in the operated/deployed applications, the following elements need to be applied over
+* log generation
+* and metric generation.
+
+## Log generation
+Applies to all application components.
+
+### Log entry best practices
+* Log entry format and text taxonomy MUST be unified across all log providers for easy human processing.
+* Log entries (in any case) MUST NOT contain Personal Identifiable Information (PII) for security reasons.
+* Each log entry MUST reflect a single event and vice-versa for determinability.
+* Log entries MUST use simple English language or widely recognised characters (when solving complex data representations).
+* Log entry messages MUST be meaningful and explain the event’s context (counter-example: “Error occurred”)
+* Log entry creation MUST NOT pass arguments to log statements whose evaluation will change object state for traceability and readability.
+* A “cross component tracing ID” MUST be included to allow tracking all generated logs across the multiple components for traceability.
+
+### What to log?
+* Requests: all incoming request to a local component.
+* Remote calls: all outgoing request to a remote service. In such requests, it is crucial to log the duration of call for later debugging, 
+just as the result of remote response (success, error).
+* Scheduled tasks: all started scheduled tasks and accordingly their end-results (error, success, interrupt).
+* Application scoped resources: resource issues including exhausted resources, exceeded capacities and connectivity issues.
+* Threats: Suspicious/malicious activities against the application.
```

Comment:
how could that be predicted here, in a "what to log" aspect - can you provide an example please?